bin/es-docker

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Run Elasticsearch and allow setting default settings via env vars
```

Comment:
> I would want all ENV variables I used to configure elastic to continue working as is with containerpilot.

Sure but there's literally only one line that doesn't work as-is: https://github.com/autopilotpattern/elasticsearch/pull/8/files/2f807599ea62bf41d3fc811586ffd0648ac40496#diff-35c179adb2b0c25802542935645540a7R37